packages/root-config/src/zuri-root-config.js

```diff
@@ -10,7 +10,7 @@ import { coreApps, allPlugins } from "./applications";
 
 coreApps.forEach(app => {
   registerApplication({
-    name: `@zuri/${app.name}`,
+    name: `@zuri${app.class}/${app.name}`,
```

Comment:
whyyyy????

Reply:
To apply styling for text align right for Arabic and Hebrew texts

Reply:
please i already made the changes please review it again I'm sorry